### ISSUE TITLE
OWL update checker 

### ISSUE THREAD

**Issue by n0tb0dy** [OpenLiveWriter/OpenLiveWriter#172]
How to check for OWL updates? Is it possible to add this in help menu?

**Comment by martinwoodward** [comment]
OWL automatically checks for updates on start-up and silently installs them next time you start Open Live Writer.  It would be good to have some UI in though to let users know that they are on the latest version, maybe have a link to the release notes on GitHub etc.  The AboutForm could do with a bit of a re-write so we should possibly include the login into that.

**Comment by robertiagar** [comment]
Silent installs are good from an UX perspective, but maybe there should be an option to notify the user. Some people might not like them...

**Comment by RobStand** [comment]
Don't you mean OLW? ;)

**Comment by hmemcpy** [comment]
I like how VS Code handles it, by showing a toast when an update is available/no new update, with an action button.
![image](https://cloud.githubusercontent.com/assets/601206/11978177/131d3ba8-a992-11e5-8ea7-de9d54ee9860.png)

**Comment by hashhar** [comment]
I tried doing this by adding some code to `UpdateManager.CheckForUpdates` and was able to launch explorer.exe if an update was available. Couldn't get further than that.

**Comment by kathweaver** [comment]
Is there an option so that OpenLiveWriter doesn't automatically update?  Working on some documentation.

**Comment by hashhar** [comment]
Nope. I don't think so. But you can opt into nighty builds. It is accessible from the registry though.

**Comment by SteveP-UK** [comment]
Hi! I'm already using the latest version but interested to know how to opt
in for 'nightly builds'?

**Comment by shanselman** [comment]
Thanks for reporting this and for helping Open Live Writer over the years. We are doing an issue-bankruptcy cleanup so we can make the repository maintainable again and focus on current, actionable work. This older issue is being closed as part of that cleanup, not because it was unimportant. If this still reproduces in the current code or is still relevant to the revitalization work, please open a fresh issue with current details and we will triage it going forward.
